spec/rack_attack_throttle_spec.rb

```diff
@@ -148,3 +148,38 @@
     end
   end
 end
+
+describe 'Rack::Attack.throttle with case sensivity option for discriminator' do
```

Comment:
Consider updating description to be more generalized, omitting the case-sensitivity text.